spec/latest/core/index.html

```diff
@@ -908,6 +912,7 @@ <h3>Description of HTTP Status Codes and Errors</h3>
       {
         "@context": "http://www.w3.org/ns/hydra/context.jsonld",
```

Comment:
I'd go with problem specific context

Reply:
I agree but we're not there yet. IMO this has to wait until there is more work done in #178

Reply:
Hmm. Is there any chance to expand this PR with additional context? Or shall we have a separate PR for that later

Reply:
I think we'll add it later. Currently there is nothing we'd put in such context

Reply:
> Well, it would have details of how RFC 7807 problem would be translated to hydra.

Like I said, we'd have to work on #178 first. The result would be that context.
Otherwise, this snippet has no relation with problem+json.

> Then, we could have it linked to hydra's main context.

Out of scope of this PR, but I think I'd have ot the opposite way